demos/python_demos/object_detection_demo_faceboxes/README.md

```diff
@@ -0,0 +1,48 @@
+# Object Detection FaceBoxes Demo
```

Comment:
is no need to say that it is face detection demo. FaceBoxes  is only face detection model

Reply:
I would keep that name. It already includes "face". And it will be simpler for us to find all demos and models to merge into one object_dtection_demo later.